app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    if command[0] == "cp" and command[1] != command[2]:
```

Comment:
you can unpack these variables in the easiest way, also use the more descriptive names
command, old_file, new_file = command.split()